test/puppetlabs/puppetdb/config_test.clj

```diff
@@ -173,6 +169,15 @@
                      (.mkdir)
                      (.setWritable true))]
       (is (= (validate-vardir (vardir filename))
+             (vardir filename)))))
+
+  (testing "should support relative paths"
+    (let [filename "./target/totally/okay"]
+      (doto (fs/file filename)
+        (.deleteOnExit)
+        (.mkdirs)
```

Comment:
I more meant you could use `(temp-dir "foo")` and use that for testing out relativity from that temp-dir. I.e. `/foo/../foo/./bar` or some such.